notificationworkerlambda/src/test/scala/com/gu/notifications/worker/delivery/apns/models/payload/ApnsPayloadBuilderSpec.scala

```diff
@@ -402,4 +405,75 @@ class ApnsPayloadBuilderSpec extends Specification with Matchers {
         |}""".stripMargin
     )
   }
+
+  trait UsElectionNotificationScope extends NotificationScope {
+    val notification = models.Us2020ResultsNotification(
+      id = UUID.fromString("068b3d2b-dc9d-482b-a1c9-bd0f5dd8ebd7"),
+      sender = "test",
+      title = Some("US elections 2020: Live results"),
+      message = Some("normal message"),
+      link = Internal("world/2016/jul/26/men-hostages-french-church-police-normandy-saint-etienne-du-rouvray", Some("https://gu.com/p/4p7xt"), GITContent, None),
```

Comment:
Are we still using these links?

Reply:
This may not be relevant to this particular PR, but sharing to make sure everyone is aware 🙏.

We should not be using `gu.com` domain links anymore.

The business in the long process of selling `gu.com`. If a shorter `url` than the canonical `url` (i.e `https://www.theguardian.com/world/2016/jul/26/men-hostages-french-church-police-normandy-saint-etienne-du-rouvray`) is absolutely needed the main domain (i.e `www.theguardian.com`) and the shortcode path (i.e  `p/4p7x` ) could be associated  (i.e `https://www.theguardian.com/p/4p7xt`).